src/Components/Search/NewSearch/Mobile/SearchResultsList.tsx

```diff
@@ -0,0 +1,247 @@
+import { ContentPlaceholder } from "Components/Search/NewSearch/Mobile/SearchResultsList/ContentPlaceholder"
+import { NoResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/NoResults"
+
+interface SearchResultsListProps {
+  relay: RelayPaginationProp
+  viewer: SearchResultsList_viewer$data
+  query: string
+  onClose: () => void
+}
+
+const ENTITIES_PER_SCROLL = 10
+
+const SearchResultsList: FC<SearchResultsListProps> = ({
+  relay,
+  viewer,
+  query,
+  onClose,
+}) => {
+  const tracking = useTracking()
+  const [isLoading, setIsLoading] = useState(false)
+
+  const options = extractNodes(viewer.searchConnection)
+
+  if (!relay.isLoading() && options.length === 0) {
+    return <NoResults query={query} mt={50} mx={20} />
+  }
+
+  tracking.trackEvent({
+    action_type:
+      options.length > 0
+        ? DeprecatedSchema.ActionType.SearchedAutosuggestWithResults
+        : DeprecatedSchema.ActionType.SearchedAutosuggestWithoutResults,
+    query: query,
+  })
+
+  const formattedOptions: SuggionItemOptionProps[] = formatOptions(
+    options as SearchNodeOption[]
+  )
+
+  const handleLoadMore = () => {
+    if (!relay.hasMore() || relay.isLoading()) {
+      return
+    }
+
+    setIsLoading(true)
+
+    relay.loadMore(ENTITIES_PER_SCROLL, err => {
+      if (err) {
+        console.error(err)
+      }
+
+      setIsLoading(false)
+    })
+  }
+
+  const handleOnRedirect = () => {
+    onClose()
+  }
+
+  return (
+    <>
+      {formattedOptions.map((option, index) => {
+        return (
+          <NewSuggestionItem
+            query={query}
+            option={option}
+            onRedirect={handleOnRedirect}
+            key={index}
+          />
+        )
+      })}
+
+      {relay.hasMore() && <InfiniteScrollSentinel onNext={handleLoadMore} />}
+
+      {!relay.hasMore() && (
+        <SeeFullResults query={query} onClick={handleOnRedirect} />
+      )}
+
+      {isLoading && (
+        <Flex width="100%" minHeight="60px" alignItems="center">
+          <Spinner position="relative" />
+        </Flex>
+      )}
+    </>
+  )
+}
+
+const SearchResultsListPaginationContainer = createPaginationContainer(
+  SearchResultsList,
+  {
+    viewer: graphql`
+      fragment SearchResultsList_viewer on Viewer
+        @argumentDefinitions(
+          first: { type: "Int", defaultValue: 10 }
+          after: { type: "String" }
+          term: { type: "String!", defaultValue: "" }
+          hasTerm: { type: "Boolean!", defaultValue: false }
+          entities: { type: "[SearchEntity]" }
+        ) {
+        searchConnection(
+          query: $term
+          entities: $entities
+          mode: AUTOSUGGEST
+          first: $first
+          after: $after
+        )
+          @include(if: $hasTerm)
+          @connection(key: "SearchResultsList_searchConnection") {
+          edges {
+            node {
+              displayLabel
+              href
+              imageUrl
```

Comment:
It doesn't look like any of these properties are being used in this component? Is there a missing fragment container?

Reply:
I pass these properties to the [formatOptions](https://github.com/artsy/force/pull/12410/files#diff-236782da27733a3ff4c1063878a65fb8dcd8e43348e55fde1e5319b57b9bd884R16) helper function.

Reply:
but will be refactored because of [this thread](https://github.com/artsy/force/pull/12410#discussion_r1195450367)